test/images/devenv/install.sh

```diff
@@ -173,6 +185,12 @@ check_install() {
     shellcheck --version
     tkn version
     yq --version
+    if [[ "$(grpc_cli help)" == *"command not found"* ]]; then 
+       echo "[ERROR] Could not find grpc_cli"
```

Comment:
Please redirect the error to stdout.

Reply:
updated now

Reply:
Did you forget to commit the change? I don't see the redirection in the diff.

Reply:
it's showing for me, but I've just done a rebase so maybe it will update it for you too